.github/workflows/_test.yml

```diff
@@ -49,6 +49,10 @@ jobs:
         with:
           python-version: ${{ inputs.python-version }}
           pip-install: ".[dev]"
+
+      - name: Run import linter
+        run: lint-imports
+
```

Comment:
This isn't needed as pre-commit runs the test on all files anyway

Reply:
ah right